docs/fidesops/docs/index.md

```diff
@@ -23,7 +23,7 @@ In a software organization, the team that writes and delivers software is normal
 
 ![Fidesops business process](img/fides-ops-process.png "Fidesops biz process")
 
-Your policies and database annotations are written in [**Fideslang**](https://github.com/ethyca/privacy-taxonomy): the syntax that describes the attributes of your data and its allowed purposes of use. 
+Your policies and database annotations are written in [**Fideslang**](https://github.com/ethyca/fideslang): the syntax that describes the attributes of your data and its allowed purposes of use. 
```

Comment:
Absolute eagle eyes sniping the redirect! Thanks

Reply:
No problem! I'm surprised I noticed it